jscomp/bin/bsb.ml

```diff
@@ -6499,7 +6499,7 @@ let rec parsing_source (dir_index : int) cwd (x : Ext_json.t String_map.t )
         let res  = 
           Array.fold_left (fun  origin json ->
               match json with 
-              | `Obj m -> 
+              | `Obj m -> (* could also be a string *)
```

Comment:
in the future, maybe we can allow user a simplified form:
```
"subdirs" : ["dir1", "dir2","dir3"]
```
instead of current
```
"subdirs" : [ {"dir: "dir1"}, {"dir": "dir2"} ]
```